packages/adena-extension/src/containers/transfer-input-container/transfer-input-container.tsx

```diff
@@ -50,7 +50,8 @@ const TransferInputContainer: React.FC = () => {
     if (!isNext()) {
       return;
     }
-    const validAddress = addressBookInput.validateAddressBookInput();
+    const validAddress = addressBookInput.validateAddressBookInput() &&
+      (isNativeTokenModel(tokenMetainfo) || addressBookInput.validateEqualAddress());
```

Comment:
`(!isNativeTokenModel(tokenMetainfo) && addressBookInput.validateEqualAddress())`
I think this is correct, or is it not? @jinoosss

Reply:
`(!isNativeTokenModel(tokenMetainfo) && addressBookInput.validateEqualAddress()) `
conditional works fine for non-native tokens, but is always false for native tokens.